docs/deployment/feature_gates.md

```diff
@@ -34,6 +34,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | RemoveAPIServerProxyLegacyPort           | `true`  | `Beta`  | `1.119` |         |
 | IstioTLSTermination                      | `false` | `Alpha` | `1.114` |         |
 | CloudProfileCapabilities                 | `false` | `Alpha` | `1.117` |         |
+| DoNotCopyBackupCredentials               | `false` | `Alpha` | `1.120` |         |
```

Comment:
We have to update the feature gate version to 1.121 now as meanwhile https://github.com/gardener/gardener/pull/12202 got merged.

```suggestion
| DoNotCopyBackupCredentials               | `false` | `Alpha` | `1.121` |         |
```